web/src/main/java/org/akaza/openclinica/web/pform/OpenRosaServices.java

```diff
@@ -445,6 +445,7 @@ public String getManifest(@Context HttpServletRequest request, @Context HttpServ
             manifest.add(userList);
         }
         MediaFile odmPayload = new MediaFile();
+        request.getSession().setAttribute("onlyMetaNoData", "onlyMetaNoData");
```

Comment:
Instead of session attribute, add a query param called "includeClinicalData" with default value set to true. And set that parameter to false from this method.